src/rgw/rgw_op.cc

```diff
@@ -488,10 +488,9 @@ int rgw_build_bucket_policies(const DoutPrefixProvider *dpp, rgw::sal::Driver* d
   if (!s->src_bucket_name.empty()) {
     std::unique_ptr<rgw::sal::Bucket> src_bucket;
     ret = driver->get_bucket(dpp, nullptr,
-			    rgw_bucket(s->src_tenant_name,
-				       s->src_bucket_name,
-				       s->bucket_instance_id),
```

Comment:
what effect did this mix-up have?

Reply:
it probably never mattered. we only set `bucket_instance_id` if the request included an "rgwx-bucket-instance" header, and we removed multisite's last use of that with the resharding changes. it had only been used for [bucket listings](https://github.com/ceph/ceph/blob/quincy/src/rgw/rgw_data_sync.cc#L3270), not copies